crates/orderbook/openapi.yml

```diff
@@ -579,11 +579,16 @@ components:
         Provides the additional data for ethflow orders
       type: object
       properties:
-        is_refunded:
+        refunded_status:
           description: |
-            Provides a flag that shows whether the refunding service has 
-            refunded the order, in case it was not settled onchain
-          type: boolean
+            Describes the status of the refund
+          type: string
+          enum: 
+            [
+              NonRefundable,
+              NotYetRefunded,
+              Refunded,
+            ]
```

Comment:
I see the `user_valid_to` do we plan to add the `user_owner`?

EDIT: I saw below this is part of the smart orders data (so no in the eth flow data). I have questions, but wrote there.